### PR TITLE
Extract search and add flow types

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/pages/ResultsListPage.js [hunk 1]
```diff
@@ -103,25 +103,14 @@ class ResultsListPage extends React.Component {
...
-            var coordinates =
-                JSON.parse(sessionstorage.getItem('coordinates'));
-            request.location =
-                `${coordinates.longitude}E${coordinates.latitude}N`;
+            coordinates = JSON.parse(sessionstorage.getItem('coordinates'));
         } catch (e) {
         }
 
-        iss('search/', request)
+        search(location, coordinates, this.search)
```
Comment (danni) [inline src/pages/ResultsListPage.js]: iss.search ?

**Comment by danni** [comment]
One niggle about name, otherwise ++. I believe there's one more place this is used?

**Comment by DanielHeath** [comment]
Nowhere else seems to call search, but there's another consumer of `iss` (I've extracted getService)

**Comment by danni** [comment]
++

**Pull request merged**